.coderabbit.yaml

```
@@ -0,0 +1,172 @@
+    - path: "**/*.json"
+      instructions:
+        "Indent size for JSON files needs to be 2.\n
+        In JSON, an array wrapping needs to be split into lines.\n
+        Trailing commas in JSON need to be removed.\n
+        Object wrapping in JSON needs to be split into lines.\n
+        Avoid alignment of properties in JSON.\n
+        Space needs to be introduced after colon and comma in JSON.\n
+        Do not introduce space before colon or comma in JSON."
+
+chat:
+  auto_reply: true
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)